HealthcareSystem/Backend/Model/Users/Doctor.cs

```diff
@@ -10,7 +11,10 @@ namespace Model.Users
    public class Doctor : User
    {
         public string NumberOfLicence { get; set; }
-        public TypeOfDoctor Type { get; set; }
+
+        [ForeignKey("Specialty")]
+        public int SpecialtyId { get; set; }
+        public virtual Specialty Specialty { get; set; }
```

Comment:
This isn't too relevant, but wouldn't making this a list of specialties make more sense? It doesn't make using it more complicated and it might be more accurate for real life, doctors aren't limited to having just one specialty.

Reply:
Applied in f48613f08e758ec1d13738d35d386e2834e035de.